SECURITY.md

```diff
@@ -1,5 +1,13 @@
 # Security Policy
 
+## Supported Versions
+
+The Strimzi project provides security vulnerability fixes on its components as part of a new patch version of the current minor release and/or on the next minor release, from the `main` branch. The choice about having a new patch version, making the fix only on the next minor release or both depends on the severity of the vulnerability.
+For example, if the latest version of the cluster operator is 0.28.0, a vulnerability fix can be part of a 0.28.1 and/or 0.29.0.
+The same applies to the other Strimzi components like the Kafka bridge, Kafka OAuth and so on.
+Regarding the Apache Kafka security vulnerabilities, their fix follow the release process in the upstream Apache Kafka community.
+A list of fixed CVEs is provided [here](https://kafka.apache.org/cve-list).
+
```

Comment:
What about keeing it a bit more simple ... something like this for example? I would also put it only after the _Reporting a Vulnerability_ section.

```suggestion
## Supported Versions

Depending on the severity of the CVE or other vulnerability, the fix will be provided either as a patch release of the current minor release or in the next minor release.
Strimzi uses Apache Kafka binaries as provided by the Apache Kafka project.
Any CVE fixes to Apache Kafka and its dependencies need to be first fixed and released in Apache Kafka itself.

```

Reply:
> Depending on the severity of the CVE or other vulnerability, the fix will be provided either as a patch release of the current minor release or in the next minor release.

I would make clear that this is about Strimzi related components (operator, bridge and so on). At same time the example I made made it clearer as well.

> Any CVE fixes to Apache Kafka and its dependencies need to be first fixed and released in Apache Kafka itself.

I guess saying "CVE fixes" here was a typo? It should be something like "Any CVE in Apache Kafka and its dependencies need to be first fixed and released by the Apache Kafka community and then used by Strimzi" ?